src/content/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent.mdx

```diff
@@ -22,128 +22,125 @@ If you don't have one already, [create a New Relic account](https://newrelic.com
 
 ## Enable browser monitoring [#enable]
 
-To enable browser: Go to **[one.newrelic.com](https://one.newrelic.com)**, select **Browser**, and then select **Add more data**. The UI gives you instructions. 
+Browser Pro+SPA is the default agent when you enable browser monitoring. This automatically gives you access to all of our browser monitoring features.
 
-Some notes about this process: 
-* It can take several minutes after enabling the agent for data to show up in New Relic. 
-* If you are doing the copy/paste method, see [Copy/paste option](#copy-paste-app) for tips. 
-* If you're having trouble, see [Troubleshooting](/docs/browser/browser-monitoring/troubleshooting/troubleshoot-your-browser-monitoring-installation).
-* After enabling, you can configure the agent on the [**App settings** page](/docs/browser/new-relic-browser/configuration/browser-app-settings-page). 
-* For other questions, see the remainder of this doc. 
+To enable browser monitoring:
 
-## Browser agent types - Lite, Pro, Pro+SPA [#agent-types]
+1. Go to **[one.newrelic.com](https://one.newrelic.com)**, select **Browser**, and then select **Add more data**.
+2. Follow the instructions in the UI.
```

Comment:
We might want to add a little guidance about what to do on that Add more data page. I assume they want to click the New Relic Browser button?